src/index.ts

```diff
@@ -200,3 +259,28 @@ export interface ErrorResultType {
   code: string;
   message: string;
 }
+
+export interface ReadProgramSpaceResultType {
+  programSpaceData: string;
+}
+
+export interface WriteProgramSpaceResultType {
+  isSuccess: boolean;
+}
+
+export enum OperationModes {
+  BEST_AVAILABLE = 'BEST_AVAILABLE',
+  FULL = 'FULL',
+}
+
+export enum Modalities {
```

Comment:
Enums should be singular as well. 
```suggestion
export enum Modality {
```

Reply:
Remember to make documentation changes for all these changes.